protocol/communities/manager.go

```diff
@@ -4003,7 +4003,7 @@ func (m *Manager) ImageToBase64(uri string) string {
 	return base64img
 }
 
-func (m *Manager) AddCommunityToken(token *CommunityToken) (*CommunityToken, error) {
+func (m *Manager) AddCommunityToken(token *CommunityToken, croppedImage *images.CroppedImage) (*CommunityToken, error) {
```

Comment:
I was considering putting the `CroppedImage` into the already provided `CommunityToken`.

However, it turns out that `CommunityToken` doesn't actually use that data. It's only interested in the base64 encoded image data, which atm is first provided via `Base64Image` as image path, which is then overridden with an actual base64 string.

So to not pollute `CommunityToken` with unnecessary data, I think it's better to keep `CroppedImage` separate.
There's an argument to be made that one could rather put these in a `request` object on the messenger level.

Lemme know what you think.

Reply:
Can you elaborate on putting in a request object? Where could it be?

Reply:
So the idea would be the `Messenger` would receive a `requests.AddCommunityToken` object that has a `CommunityToken` and ` CroppedImage`, similar to how we do `request` objects in all other messenger APIs.

Manager would still receive separate arguments I think.